public/stylesheets/overlays/generic/generic.css

```diff
@@ -0,0 +1,647 @@
+        /* Right game edge */ 1180px 20px, 
+        /* Title */ 
+        1040px 20px, 1034px 36px, 830px 36px, 
+        814px 60px, 386px 60px, 370px 36px, 
+        166px 36px, 160px 20px, 20px 20px
+    );
+}
+
+.game16by9-layout-1cam {
+    -webkit-clip-path: polygon(
+        /* Outer border */
+        0px 0px, 1920px 0px, 1920px 1080px, 0px 1080px, 0px 0px, 192px 10px, 192px 794px,
+        /* Left name */ 212px 814px, 632px 814px, 652px 854px, 662px 854px, 
+        /* Left score */ 642px 814px, 722px 814px, 742px 854px, 
+        /* Player cam */ 
+        960px 854px, 960px 864px, 768px 864px, 768px 1070px, 
+        1152px 1070px, 1152px 864px, 960px 864px, 960px 854px, 1178px 854px,
+        /* Right score */ 1198px 814px, 1278px 814px, 1258px 854px, 
+        /* Right name */ 1268px 854px, 1288px 814px, 1708px 814px, 1728px 794px, 
+        /* Tourney info */ 1728px 10px, 1536px 10px, 1516px 60px, 1096px 60px, 1076px 30px,
+        /* Logo space */ 844px 30px,
+        /* Title */ 824px 60px, 404px 60px, 384px 10px, 192px 10px
+    );
+}
+
+.all16by9-layout-2cam {
+    -webkit-clip-path: polygon(
+        /* Outer border */
+        0px 0px, 1920px 0px, 1920px 1080px, 0px 1080px, 0px 0px, 10px 10px, 10px 710px,
+        /* Commentator cam */ 1910px 710px, 1910px 365px, 1280px 365px, 
+        /* Cam separator */ 1280px 355px, 
+        /* Player cam */ 1910px 355px, 1910px 10px, 1280px 10px, 1280px 710px, 1270px 710px,
+        /* Right game edge */ 1270px 10px, 
+        1130px 10px, 1122px 30px, 870px 30px, 
+        850px 60px, 430px 60px, 410px 30px, 
+        158px 30px, 150px 10px, 10px 10px
+    );
+}
+
+.container {
+    width: 1920px;
+    height: 1080px;
+    position: absolute;
+    left: 0;
+    top: 0;
+}
+
+#background {
+    background-image: url(http://i.imgur.com/rL7QNS0.jpg);  
+}
+
+#main-container {
+    -webkit-clip-path: none;
+}
+
+/* empty-layout */
+
+    #main-container.empty-layout .playerText {
+        position: absolute;
+        color: white;
+        font-size: 42px;
+        font-family: 'Oswald', cursive;
+        text-shadow: 2px 2px 2px rgb(50, 50, 50);
+
+        /* Oswald sits a little low, so raise the baseline */
+        margin-top: -5px;
+
+        overflow: hidden;
+        overflow-wrap: break-word;
+    }
+
+    #main-container.empty-layout .title,
+    #main-container.empty-layout .tourneyInfo,
+    #main-container.empty-layout .commentators,
+    #main-container.empty-layout .lCharacter,
+    #main-container.empty-layout .rCharacter,
+    #main-container.empty-layout .lIcons,
+    #main-container.empty-layout .lState, 
+    #main-container.empty-layout .lPort,    
+    #main-container.empty-layout .lPlayer, 
+    #main-container.empty-layout .lScore, 
+    #main-container.empty-layout .rScore, 
+    #main-container.empty-layout .rPlayer, 
+    #main-container.empty-layout .rPort, 
+    #main-container.empty-layout .rState, 
+    #main-container.empty-layout .rIcons  {
```

Comment:
This looks gross. We should either
A: Apply this to `#main-container.empty-layout`
or
B: Add another class to these elements so they can all be selected with one selector